src/Utility.js

```diff
@@ -3,9 +3,11 @@ export const online = false
 
 export function getAllCubsPlayers() {
     if (online) {
```

Comment:
Passing in a variable so that I can flip back and forth between live API data and mock data ... since I only have 1000 API calls each month.